portfolio/src/main/webapp/style.css

```diff
@@ -1,10 +1,92 @@
 body {
```

Comment:
Generally I think this CSS won't take effect that way you expect: tt will work if you load the HTML directly in an editor but because of how the app engine container resolves paths it won't be found when you actually run things in app engine. I think you'll need to add a leading `/` to both your style.css and your script.js when they're referenced in index.html.